### PR TITLE
Update raw config

### PR DESCRIPTION
**Pull request by sedroche**
# Motivation

Ticket: https://issues.jboss.org/browse/RHMAP-10900

Connected to fheng/fh-supercore/pull/889 we need a way to update the raw config for a running supercore.

# Solution

Exporting the needed function.

@wei-lee @pb82 @rachael-oregan

### CODE REVIEW

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: lib/config.js [hunk 1]
```diff
@@ -119,14 +119,31 @@ function reload(workers, cb) {
         worker.process.kill(RELOAD_CONFIG_SIGNAL);
       }
     } else {
-      rawConfig = newConfig.rawConfig;
-      fhConfig.reload(rawConfig);
+      updateRawConfig(newConfig);
```
Comment (wei-lee) [inline lib/config.js]: I don't really think this function is being used. I suggest we either remove it (we can bump the major version), or mark it as deprecated and remove it in the future.

Comment (sedroche) [reply lib/config.js]: Have removed this function and bumped major version.

File: lib/config.js [hunk 2]
```diff
@@ -105,25 +105,16 @@ function loadConfig(cb) {
   });
 }
 
-function reload(workers, cb) {
+function reloadRawConfig(cb) {
```
Comment (wei-lee) [inline lib/config.js:107]: should we add a test for this function?

Comment (sedroche) [reply lib/config.js:107]: Indeed. Will add now.

**Review by wei-lee — APPROVED** [review]
LGTM

**Pull request merged**